tools/benchmark_tool/setup.py

```diff
@@ -21,6 +23,13 @@
         in pkg_resources.parse_requirements(requirements_txt)
     ]
 
+try:
+    importlib.import_module('cv2')
+except ImportError as opencv_import_error:
+    warnings.warn(
+            "Problem with cv2 import: \n{}\n opencv-python-headless will be added to requirements".format(opencv_import_error)
+    )
+    reqs.append('opencv-python-headless==4.5.*')
```

Comment:
Why don't we support the latest stable version 4.6?

Reply:
I do not know, I'm not benchmark app developer :) 
I just replace one lib to another with keeping version as was used before

Reply:
I think you can check it with the latest version and relax upper bound

Reply:
I do not want mixup dependencies management and resolving issue with running on headless platform, the first one is not my responsibility.
If you need it, then please create ticket

Reply:
But if you need my opinion, I would say that here should not be binding specific opencv version at all for benchmark app (only basic functionality used - read image, resize with linear interpolation, which not affected by new releases).
Anyway this does not help me to pass precommit... :(

Reply:
What is with pre-commit?